config/field_test.yml

```diff
@@ -0,0 +1,25 @@
+experiments:
+  user_home_feed: # Home feed collection for logged in user
+    variants:
+      - base
```

Comment:
These are just placeholder names. Naming them probably makes sense as opposed to "a", "b" "c" eh?

We also probably want to settle on the rules of whether these are immutable or mutable, e.g. if we have one with some purpose, do we change it on an ongoing basis based on the hypotheses or do we have an append-only approach to the variations?

Reply:
For recording outcomes, I'd argue changing the contents of an experiment option is baaaaaad.